### ISSUE TITLE
Accessibility: no labels on enrollment page

### ISSUE THREAD

**Issue by deinhofer** [studyathome-internationally/hub#9]
Some form elements don't have label:

![image](https://user-images.githubusercontent.com/4621810/87404944-192c9900-c5bf-11ea-99f7-b29b3b43280a.png)

![image](https://user-images.githubusercontent.com/4621810/87404965-1fbb1080-c5bf-11ea-8b1e-a90903b360d1.png)
